app/Listeners/QueueMessage.php

```diff
@@ -33,20 +33,18 @@ public function handle(QueueMessageRequest $event)
         $resources = $event->resources;
 
         // Build the email.
-        $email = new Email($resources['message'], $resources['contest'], $resources['competition'], $resources['users']);
-
-        foreach ($email->allMessages as $message) {
-            $content = $message['message'];
+        $email = new Email($resources);
```

Comment:
If you've manually created a named binding for the manager in your AppServiceProvider, you could do that. I kinda like using the full class path since it's more explicit about what is being created & doesn't require any extra setup.